### PR TITLE
AAU, on Documentation/github, I see an end-to-end getting started rec…

### PR DESCRIPTION
**Pull request by mDuval1**
…ipe for "Object Detection task"

- [x] I opened a merge request on `kili` on a branch with the same name than the branch of the current pull request and forced security tests

### CODE REVIEW

File: recipes/getting-started/getting_started-classification.ipynb [hunk 1]
```diff
@@ -0,0 +1,624 @@
...
+    "### List of projects"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You arrive on a list of projects.\n",
+    "\n",
+    "![](../img/project_list.png)\n",
```
Comment (marcenacp) [inline recipes/getting-started/getting_started-classification.ipynb:63]: Remove email from screenshot

File: recipes/getting-started/getting_started-classification.ipynb [hunk 2]
```diff
@@ -0,0 +1,624 @@
...
+   "metadata": {},
+   "source": [
+    "### Access your project from the API"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When you ran the [command](#command) to create a project, it output a unique identifier of the project. This identifier is used to recognize, access and modify the project from the API\n",
```
Comment (marcenacp) [inline recipes/getting-started/getting_started-classification.ipynb:186]: `it outputs`

File: recipes/getting-started/getting_started-classification.ipynb [hunk 3]
```diff
@@ -0,0 +1,624 @@
...
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "kauth = KiliAuth(\n",
+    "    api_key='f09dc2c0-2c43-4870-8e4f-30758188933',\n",
```
Comment (marcenacp) [inline recipes/getting-started/getting_started-classification.ipynb:417]: Remove

File: recipes/getting-started/getting_started-classification.ipynb [hunk 4]
```diff
@@ -0,0 +1,624 @@
...
+    "    api_endpoint=\"http://localhost:4001/api/label/v2/graphql\")\n",
```
Comment (marcenacp) [inline recipes/getting-started/getting_started-classification.ipynb:418]: Use an environment variable

File: recipes/getting-started/getting_started-classification.ipynb (hunk 3 above)
Comment (marcenacp) [reply recipes/getting-started/getting_started-classification.ipynb:417]: => Use an environment variable

File: recipes/getting-started/getting_started-classification.ipynb [hunk 5]
```diff
@@ -0,0 +1,624 @@
...
+    "[\n",
+    "  {\n",
+    "    \"content\": \"https://cloud.kili-technology.com/api/label/v2/files?id=f436f198-cede-4380-a119-f5d827f8a8fa\",\n",
+    "    \"externalId\": \"car_2.jpg\",\n",
+    "    \"id\": \"ckm0ligy900uuc49k1idydxsk\",\n",
+    "    \"jsonMetadata\": {},\n",
+    "    \"labels\": [\n",
+    "      {\n",
+    "        \"author\": {\n",
+    "          \"email\": \"test+admin@kili-technology.com\",\n",
```
Comment (marcenacp) [inline recipes/getting-started/getting_started-classification.ipynb:365]: Obfuscate

File: recipes/getting-started/getting_started-classification.ipynb [hunk 6]
```diff
@@ -0,0 +1,624 @@
...
+    "          \"id\": \"ckm0iugnh002rc49khkrabqdd\",\n",
```
Comment (marcenacp) [inline recipes/getting-started/getting_started-classification.ipynb:366]: Obfuscate

File: recipes/getting-started/getting_started-classification.ipynb (hunk 1 above)
Comment (mDuval1) [reply recipes/getting-started/getting_started-classification.ipynb:63]: Done

File: recipes/getting-started/getting_started-classification.ipynb (hunk 2 above)
Comment (mDuval1) [reply recipes/getting-started/getting_started-classification.ipynb:186]: Done

File: recipes/getting-started/getting_started-classification.ipynb (hunk 3 above)
Comment (mDuval1) [reply recipes/getting-started/getting_started-classification.ipynb:417]: Done

File: recipes/getting-started/getting_started-classification.ipynb (hunk 4 above)
Comment (mDuval1) [reply recipes/getting-started/getting_started-classification.ipynb:418]: Removed

File: recipes/getting-started/getting_started-classification.ipynb (hunk 6 above)
Comment (mDuval1) [reply recipes/getting-started/getting_started-classification.ipynb:366]: Done

**Review by marcenacp — APPROVED** [review]

**Pull request merged**